--- kdebindings-4.2.3/csharp/qyoto/src/qyotosmokebinding.cpp.orig	2009-05-10 20:20:41.000000000 +0200
+++ kdebindings-4.2.3/csharp/qyoto/src/qyotosmokebinding.cpp	2009-05-10 20:20:10.000000000 +0200
@@ -13,12 +13,12 @@
  *                                                                         *
  ***************************************************************************/
 
-#include <stdlib.h>
 #include "qyotosmokebinding.h"
 #include "qyoto.h"
 #include "virtualmethodcall.h"
 
 #include <cstdlib>
+#include <cstdio>
 
 #include <qt_smoke.h>
 
--- kdebindings-4.2.3/csharp/plasma/src/plasmahandlers.cpp.orig	2009-05-10 20:17:41.000000000 +0200
+++ kdebindings-4.2.3/csharp/plasma/src/plasmahandlers.cpp	2009-05-10 20:18:35.000000000 +0200
@@ -15,6 +15,8 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <cstdio>
+
 #include <qyoto.h>
 #include <smokeqyoto.h>
 #include <marshall_macros.h>
